projects/packages/videopress/src/client/block-editor/blocks/video/components/privacy-and-rating-panel/index.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ *External dependencies
+ */
+import { PanelBody, SelectControl } from '@wordpress/components';
+import { __, _x } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import {
+	VIDEO_PRIVACY_LEVELS,
+	VIDEO_PRIVACY_LEVEL_PRIVATE,
+	VIDEO_PRIVACY_LEVEL_PUBLIC,
+	VIDEO_PRIVACY_LEVEL_SITE_DEFAULT,
+} from '../../../../../state/constants';
+import { VideoControlProps } from '../../types';
+import type React from 'react';
+
+/**
+ * Sidebar Control component.
+ *
+ * @param {VideoControlProps} props - Component props.
+ * @returns {React.ReactElement}    Component template
+ */
+export default function PrivacyAndRatingPanel( { attributes, setAttributes }: VideoControlProps ) {
+	const { privacySetting } = attributes;
+
+	return (
+		<PanelBody title={ __( 'Privacy and rating', 'jetpack-videopress-pkg' ) } initialOpen={ false }>
```

Comment:
Checking other blocks [[#](https://wordpress.com/support/wordpress-editor/blocks/blog-posts-block/#block-settings)], [[#](https://wordpress.com/support/wordpress-editor/blocks/video-block/#block-toolbar)], maybe we should put the first letter in caps.
```suggestion
		<PanelBody title={ __( 'Privacy and Rating', 'jetpack-videopress-pkg' ) } initialOpen={ false }>
```

Reply:
it was explicitly requested as that. pxWta-1zy-p2